app/models/vim_performance_analysis.rb

```diff
@@ -283,9 +281,6 @@ def get_vm_needs
       if VimPerformanceAnalysis.needs_perf_data?(options[:vm_options])
         # Get VM performance data
         start_time, end_time = get_time_range(options[:range])
-        # Set :only_cols for VM daily requested cols for better performance
-        options[:ext_options] ||= {}
```

Comment:
`options[:ext_options]` is passed to `VimPerformanceAnalysis.get_daily_perf` a few lines below.

why is this initializer being removed?